tests/screenshots.test.ts

```diff
@@ -40,6 +40,25 @@ function dark(...parts: string[]): string {
   return colors.bgHex("#000").hex("#FFF")(parts.join(""));
 }
 
+defineTest("colorScheme: dark", {
```

Comment:
I suggest:

- Using `kabab-case` in test names like existing tests for uniformity, as characters like colons `:` are not supported on other file systems like Windows.
- Putting the new tests above with the existing ones, instead of in utils here.